scripts/editor/attributes.js

```diff
@@ -146,12 +146,18 @@ export const checkAttr = (key, attributes, manifest, undefinedAllowed = false) =
 	// Check current component attributes.
 	const manifestKey = manifest.attributes[key];
 
+	let tipOutput = '';
+
+	if ('components' in manifest) {
+		tipOutput = ' If you are using additional components, check if you used the correct block/component prefix in your attribute name.';
```

Comment:
Is the blank space at the beginning of the string intentional?

Reply:
@piqusy probably, as couple of lines below you can find
```
... nent manifest.${tipOutput} Please check ...
```